src/m3ninx/.gitignore

```diff
@@ -0,0 +1,24 @@
+*.out
```

Comment:
Probably remove this and to just inherit the monorepo-level `.gitignore`.

Reply:
done